src/openrct2/scenario/Scenario.h

```diff
@@ -344,7 +344,9 @@ enum
     OBJECTIVE_10_ROLLERCOASTERS_LENGTH,
     OBJECTIVE_FINISH_5_ROLLERCOASTERS,
     OBJECTIVE_REPLAY_LOAN_AND_PARK_VALUE,
-    OBJECTIVE_MONTHLY_FOOD_INCOME
+    OBJECTIVE_MONTHLY_FOOD_INCOME,
+
+    OBJECTIVE_COUNT
 };
```

Comment:
Would it be too annoying to do the enum class refactor of the objectives in this PR?

Reply:
I prefer doing that in a separate PR to keep things simple.

Reply:
Okay